projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +149,19 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function verifyCreateTicketPopupIsClosed(){
+  cy.get('cx-customer-ticketing-create-dialog').should('not.exist');
+}
+
+export function verifyFieldInputDoesntExceedLimit(){
+  cy.get('cx-customer-ticketing-create-dialog').within(() => {
+    cy.get('textarea').first().invoke('text').then((text) => {
+      expect(text.length).to.be.lessThan(256);
```

Comment:
```suggestion
      expect(text.length).to.be.lessThan(TICKET_SUBJECT_MAX_LENGTH + 1);
```
Even better if you have `lessOrEqualsThan`